.travis.yml

```diff
@@ -2,28 +2,27 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
+php: [5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.3
+    - php: 7.1
       env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
+    - php: 5.4
       env: DEPENDENCIES=dev
 
 cache:
   directories:
     - $HOME/.composer/cache/files
```

Comment:
you should also ensure that the PHPUnit install performed by the bridge is cached whenever possible. See https://github.com/KnpLabs/KnpMenu/commit/170f8d7530d263116bd72b8047014494bc187a05 for an example